loom/script/native/lsNativeInterface.cpp

```diff
@@ -26,6 +26,8 @@
 #include "loom/script/reflection/lsPropertyInfo.h"
 #include "loom/common/core/allocator.h"
 
+lmDefineLogGroup(nativeInterfaceLogGroup, "script.itf", 1, LoomLogInfo);
```

Comment:
It is kinda ugly. Hopefully you never see the messages from this and above anyway though :)